tornjak-frontend/src/components/entry-create.tsx

```diff
@@ -17,9 +17,85 @@ import {
   serverInfoUpdateFunc,
   agentworkloadSelectorInfoFunc
 } from 'redux/actions';
+import {
+  EntriesList,
+  AgentsList,
+  AgentsWorkLoadAttestorInfo,
+  ServerInfo,
+  TornjakServerInfo,
+} from './types';
+import { RootState } from 'redux/reducers';
+// import PropTypes from "prop-types";
+
+type CreateEntryProp = {
+  // dispatches a payload for the server selected and has a return type of void
+  serverSelectedFunc: (globalServerSelected: string) => void,
+  // dispatches a payload for list of agents with their metadata info as an array of AgentListType and has a return type of void
+  agentsListUpdateFunc: (globalAgentsList: AgentsList[]) => void,
+  // dispatches a payload for the tornjak server info of the selected server and has a return type of void
+  tornjakServerInfoUpdateFunc: (globalTornjakServerInfo: TornjakServerInfo) => void,
+  // dispatches a payload for the server trust domain and nodeAttestorPlugin as a ServerInfoType and has a return type of void
+  serverInfoUpdateFunc: (globalServerInfo: ServerInfo) => void,
+  // dispatches a payload for list of entries with their metadata info as an array of EntriesListType and has a return type of void
+  entriesListUpdateFunc: (globalEntriesList: EntriesList[]) => void,
+  // dispatches a payload for list of available selectors and their options as an object and has a return type of void
+  selectorInfoFunc: (globalSelectorInfo: { [index: string]: { label: string }[] }) => void,
```

Comment:
create type for this, or if there's a case for generics, maybe define something for it

Reply:
Done